apps/src/netsim/NetSimVizEntity.js

```diff
@@ -27,66 +27,18 @@ var tweens = require('./tweens');
  * rather than an exact representation of that entity.  Likewise, a VizEntity
  * will outlive its actual entity, because it can have a 'death' animation.
  *
- * Every VizEntity has a root element which is a <g> tag, an SVG "group"
- * that contains the other components that will actually draw.
- *
  * @constructor
  * @param {NetSimEntity} entity - the netsim Entity that this element represents
  */
 var NetSimVizEntity = module.exports =  function (entity) {
```

Comment:
[`NetSimEntity`](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/netsim/NetSimEntity.js) already exists as the base class for all simulation elements that can exist in remote storage, like NetSimNode, NetSimWire, etc.  `NetSimVizEntity` is intended to be a visualization element that maps to one of those.

When I started needing visualization elements that -didn't- map to simulation entities, I broke out 99% of this class' behavior into `NetSimVizElement`.  Now, a `NetSimVizEntity` is a `NetSimVizElement` that also maps to a simulation entity (so it has an ID that matches its corresponding simulation entity's ID).